allensdk/brain_observatory/behavior/data_objects/cell_specimens/traces/corrected_fluorescence_traces.py

```diff
@@ -48,9 +48,9 @@ def from_nwb(cls, nwbfile: NWBFile) \
 
     @classmethod
     def from_data_file(cls,
-                       demix_file: DemixFile) \
+                       neuropil_corrected_file: NeuropilCorrectedFile) \
             -> "CorrectedFluorescenceTraces":
-        corrected_fluorescence_traces = demix_file.data
+        corrected_fluorescence_traces = neuropil_corrected_file.data
```

Comment:
It should be okay here to return the single column of the neuropil corrected traces as in `neuropil_corrected_file.data.corrected_fluorescence`

Reply:
Adding .corrected_fluorescence triggers an error when the `CorrectedFluorescenceTraces` is passed into `_validate_traces` since the method selects the `corrected_fluorescence` column of a traces dataframe, but adding .correct_fluorscence gets the pd.Series from the .DataFrame and doesn't contain a named column `corrected_fluorescence`

Reply:
I just converted the series to a dataframe and it's fine now

Reply:
Okay, we can figure out how to add them on the next ticket which will likely involve more modifications this file and how they are loaded by cell specimens. There is a suggestion on that ticket.